### PR TITLE
fixing broken link

### PR DESCRIPTION
**Pull request by spelluru**

### CODE REVIEW

**Review by jsquire — CHANGES_REQUESTED** [review]
@spelluru:  This content is automatically generated from the Azure SDK repository README.  These must remain absolute links in order to ensure they work on both platforms.

File: api/overview/azure/latest/event-hubs.md [hunk 1]
```diff
@@ -6,43 +6,43 @@ ms.author: pallavit
...
+| [Microsoft.Azure.EventHubs](https://www.nuget.org/packages/Microsoft.Azure.EventHubs)  |  [API Reference for Microsoft.Azure.EventHubs](/dotnet/api/overview/azure/eventhubs/v4)  |  [Samples for Azure.Messaging.EventHubs](https://github.com/Azure/azure-event-hubs/tree/master/samples/DotNet/)  |
+| [Microsoft.Azure.EventHubs.Processor](https://www.nuget.org/packages/Microsoft.Azure.EventHubs.Processor)  |  [API Reference for Microsoft.Azure.EventHubs.Processor](/dotnet/api/microsoft.azure.eventhubs.processor)  |  [Samples for Azure.Messaging.EventHubs.Processor](https://github.com/Azure/azure-event-hubs/tree/master/samples/DotNet/)  |
+
+## Libraries for resource management
+
+Use the following library to work with the Azure Event Hubs resource provider:
+
+| NuGet Package | Reference |
+|--------------------------------------|---------------------------------------------------------------|
+| [Azure.ResourceManager.EventHubs](https://www.nuget.org/packages/Azure.ResourceManager.EventHubs)  | [API Reference for Azure.ResourceManager.EventHubs](/dotnet/api/overview/azure/resourcemanager.eventhubs-readme)  |
```
Comment (jsquire) [inline api/overview/azure/latest/event-hubs.md:48]: This link does not exist in the [source](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/README.md).   How is this getting translated?

//cc: @JimSuplizio, @danieljurek

File: api/overview/azure/latest/event-hubs.md [hunk 2]
```diff
@@ -6,43 +6,43 @@ ms.author: pallavit
 ms.data: 04/11/2023
 ms.topic: reference
 ms.devlang: dotnet
-ms.service: eventhubs
+ms.service: event-hubs
```
Comment (jsquire) [inline api/overview/azure/latest/event-hubs.md:9]: Not sure where the font matter comes from.    @JimSuplizio, @danieljurek:  Can you assist?

Comment (danieljurek) [reply api/overview/azure/latest/event-hubs.md:9]: This file doesn't end with `-readme.md` . It's a service-level overview. Changes here should not be overwritten on publish.

File: api/overview/azure/latest/event-hubs.md (hunk 1 above)
Comment (danieljurek) [reply api/overview/azure/latest/event-hubs.md:48]: These tables are hand-produced and need to be updated manually by the team. We do produce and update a list of packages associated with the service here: https://github.com/Azure/azure-docs-sdk-dotnet/blob/main/api/overview/azure/latest/event-hubs-index.md 


That can be included in this overview using an `[!INCLUDE...` directive like the one here: https://github.com/Azure/azure-docs-sdk-dotnet/blob/main/api/overview/azure/latest/active-directory.md?plain=1

File: api/overview/azure/latest/event-hubs.md (hunk 2 above)
Comment (danieljurek) [reply api/overview/azure/latest/event-hubs.md:9]: On closer inspection our service level readme automation does do special work here. Let me dig in more and see if my previous statement is still correct.

Comment (danieljurek) [reply api/overview/azure/latest/event-hubs.md:9]: The service level overview automation derives (or selects the `MSDocService`) from the first package it finds associated with that service. 

I recommend updating all package metadata for relevant service packages in the [metadata CSV file](https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/dotnet-packages.csv) to ensure that you have the right value @spelluru

**Pull request closed without merging**